haproxy/README.md

```diff
@@ -270,16 +270,21 @@ Returns `CRITICAL` for a given service if HAProxy is reporting it `down`.
 Returns `OK` for `maint`, `ok` and any other state.
 
 ## Troubleshooting
+### Port 514 Already in Use Error
+On systems with syslog, if the Agent is listening for HAProxy logs on port 514, the following error can appear in the Agent logs: 
+`Can't start UDP forwarder on port 514: listen udp :514: bind: address already in use `. 
 
-Need help? Contact [Datadog support][11].
+This is happening because by default, syslog is listening on port 514. To resolve this error, syslog can be disabled, or HAProxy can be configured to forward logs to port 514 and another port the agent is listening for logs on. The port the agent listens on can be defined in the haproxy.d/conf.yaml file [here][11]
```

Comment:
```suggestion
This is happening because, by default, syslog is listening on port 514. To resolve this error, syslog can be disabled, or HAProxy can be configured to forward logs to port 514 and another port the Agent is listening for logs on. The port the Agent listens on can be defined in the haproxy.d/conf.yaml file [here][11]
```